tests/fenv_test.cpp

```diff
@@ -181,7 +181,7 @@ TEST(fenv, fedisableexcept_fegetexcept) {
 }
 
 TEST(fenv, feenableexcept_fegetexcept) {
-#if defined(__aarch64__) || defined(__arm__)
+#if defined(__aarch64__) || defined(__arm__) || defined(__riscv)
```

Comment:
I suggest we sync the changes from RVI upstream, FE_DENORMAL is not defined for riscv.